testlib/src/test/java/com/diffplug/spotless/generic/IndentStepTest.java

```diff
@@ -82,4 +82,19 @@ protected FormatterStep create() {
 			}
 		}.testEquals();
 	}
+
+	@Test
+	void allowLeadingSpaceIfMultiLineComment() {
+		FormatterStep indent = IndentStep.Type.TAB.create(4);
+		Assertions.assertAll("class multi-line comments",
+			() -> Assertions.assertEquals(" * test", indent.format("* test", new File(""))),
+			() -> Assertions.assertEquals(" * test", indent.format(" * test", new File(""))),
```

Comment:
Try [`StepHarness`](https://github.com/diffplug/spotless/blob/main/testlib/src/main/java/com/diffplug/spotless/StepHarness.java), and make sure that the extra space is only preserved, never inserted. Something along these lines:

```java
StepHarness.forStep(IndentStep.Type.TAB.create(4))
  .testUnaffected("* test")
  .testUnaffected(" * test")
  .testUnaffected("\t* test")
  .test("    * test", "\t* test")
```